src/amo/pages/UserProfileEdit/index.js

```diff
@@ -542,8 +543,66 @@ export class UserProfileEditBase extends React.Component<InternalProps, State> {
     const userProfileURL = `/user/${userId}/`;
     const overlayClassName = 'UserProfileEdit-deletion-modal';
 
+    const bannerText = isMzaBranding()
+      ? i18n.gettext(
+          '%(boldStart)sFirefox Accounts was renamed to Mozilla accounts on Nov 1%(boldEnd)s %(newLine)s %(newLineEnd)sYou will still sign in with the same username and password, and there are no other changes to the products that you use. %(linkStart)sLearn More%(linkEnd)s',
+        )
+      : i18n.gettext(
+          '%(boldStart)sFirefox Accounts will be renamed to Mozilla accounts on Nov 1%(boldEnd)s %(newLine)s %(newLineEnd)sYou will still sign in with the same username and password, and there are no other changes to the products that you use. %(linkStart)sLearn More%(linkEnd)s',
+        );
+
+    const replacements = [
+      [
+        'linkStart',
+        'linkEnd',
+        (text) => {
+          return (
+            <a
+              target="_blank"
+              href="https://support.mozilla.org/kb/firefox-accounts-renamed-mozilla-accounts"
+              rel="noreferrer"
+              className="fxa-info-link"
+            >
+              {text}
+            </a>
+          );
+        },
+      ],
+      [
+        'newLine',
+        'newLineEnd',
+        () => {
+          return <br />;
+        },
+      ],
+      [
+        'boldStart',
+        'boldEnd',
+        (text) => {
+          return <strong>{text}</strong>;
+        },
+      ],
+    ];
+
+    const bannerContent = replaceStringsWithJSX({
+      text: i18n.sprintf(bannerText, {
+        linkEnd: '%(linkEnd)s',
+        linkStart: '%(linkStart)s',
+        newLine: '%(newLine)s',
+        newLineEnd: '%(newLineEnd)s',
+        boldStart: '%(boldStart)s',
+        boldEnd: '%(boldEnd)s',
+      }),
+      replacements,
+    });
+
     return (
       <Page>
+        <div className="warning-container">
+          <Notice type="warning" key="MzA-branding" className="MzA-branding">
```

Comment:
nit: the className isn't consistently named with other classes on the page.  It'd be something like `UserProfileEdit-MzA-banner`

Reply:
another nit: do you need `key`?  It's not in a list.

Reply:
well, now it is!